modules/tensor_mechanics/src/userobjects/CrackFrontDefinition.C

```diff
@@ -81,6 +81,12 @@ addCrackFrontDefinitionParams(InputParameters & params)
                              q_function_type,
                              "The method used to define the integration domain. Options are: " +
                                  q_function_type.getRawNames());
+  params.addParam<UserObjectName>(
+      "crack_front_points_privider",
```

Comment:
privider->provider

Reply:
Even though the name of this base class is CrackFrontPointProvider, in practice, I think the only way we're going to use it is if it is an XFEM GeometricCutUserObject. I'm kind of torn about what the name of this parameter should be. Leave it as-is, but we may want to change that.